src/apps/bridge/sensor_drivers/aanderaaConductivitySensor.cpp

```
@@ -0,0 +1,313 @@
+/**
+ * @file aanderaaConductivitySensor.cpp
+ * @brief Implementation of Aanderaa Conductivity Sensor Bridge Driver
+ *
+ * This file implements the bridge-side functionality for handling Aanderaa
+ * conductivity sensor data. It manages CBOR message reception, statistical
+ * aggregation, and integration with the bridge reporting system.
+ */
+#include "FreeRTOS.h"
+#include "aanderaaConductivitySensor.h"
+#include "aanderaa_conductivity_msg.h"
+#include "app_config.h"
+#include "app_util.h"
+#include "avgSampler.h"
+#include "bridgeLog.h"
+#include "cbor.h"
+#include "device_info.h"
+#include "pubsub.h"
+#include "semphr.h"
+#include "spotter.h"
+#include "stm32_rtc.h"
+#include "topology_sampler.h"
+#include <new>
+
+
+bool AanderaaConductivitySensor::subscribe() {
+  bool rval = false;
```

TODO: As a follow-up, I'd like to extract this pattern to the base class. Much of this code is duplicated across all sensors. We can make a struct that the base class can handle. This would apply to nearly all functions: 
* subscribe
* aanderaaConductivitySubCallback
* aggregate
* createAanderaaConductivitySub --> createSensorSub

And, truly, almost none of that is needed if we go config-driven generated code!

Added note to https://app.shortcut.com/sofar/story/243503/conductivity-sensor-follow-up-bridge

---

I know in C++ there are no issues with having "open" function parameters in a function. But the C developer in me is screaming, could we use `void` for functions with no parameters.

---

yeah, you are right. This is not portable even for C++. This can cause a silent failure/crash. Good catch!